stat/stat_test.go

```diff
@@ -1374,65 +1377,22 @@ func TestCDF(t *testing.T) {
 		}
 		for j, q := range test.q {
 			for k, kind := range cumulantKinds {
-				v := CDF(q, kind, test.x, test.weights)
-				if !floats.Equal(copyX, test.x) && !math.IsNaN(v) {
-					t.Errorf("x changed for case %d kind %d percentile %v", i, k, q)
-				}
-				if !floats.Equal(copyW, test.weights) {
-					t.Errorf("x changed for case %d kind %d percentile %v", i, k, q)
-				}
-				if v != test.ans[k][j] && !(math.IsNaN(v) && math.IsNaN(test.ans[k][j])) {
-					t.Errorf("mismatch case %d kind %d percentile %v. Expected: %v, found: %v", i, k, q, test.ans[k][j], v)
-				}
+				t.Run(test.desc, func(t *testing.T) {
+					v := CDF(q, kind, test.x, test.weights)
+					if !floats.Equal(copyX, test.x) && !math.IsNaN(v) {
+						t.Errorf("x changed for case %d kind %d percentile %v", i, k, q)
+					}
+					if !floats.Equal(copyW, test.weights) {
+						t.Errorf("x changed for case %d kind %d percentile %v", i, k, q)
+					}
+					if v != test.ans[k][j] && !(math.IsNaN(v) && math.IsNaN(test.ans[k][j])) {
+						t.Errorf("mismatch case %d kind %d percentile %v. Expected: %v, found: %v", i, k, q, test.ans[k][j], v)
+					}
+				})
 			}
 		}
 	}
 
-	// these test cases should all result in a panic
```

Comment:
Why are these removed?

Reply:
Sorry. This was mistakenly removed. I will fix it in the pull request.